engine/src/cmd/collide2/Stdafx.h

```diff
@@ -6,6 +6,8 @@
  */
 ///////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+#pragma once
```

Comment:
1. I'd rather remove `StdAfx.h`...it's a windows oriented files namely for pre-compiled headers which are themselves issues....that aside...
2. Please use the other header guard method:
```
#ifndef SOME_CONSTANT_H__
#define SOME_CONSTANT_H__
...
#endif // SOME_CONSTANT_H__ 
```
It's far more reliable and easier to debug than `#pragma once`.

Reply:
#2 Done